R/Data-methods.R

```diff
@@ -415,6 +417,84 @@ setMethod(
   }
 )
 
+# DataDA-update ----
+
+#' Updating `DataDA` Objects
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' A method that updates existing [`DataDA`] object with new data.
+#'
+#' @note This function is capable of not only adding new patients but also
+#'   updates existing ones with respect to `y`, `t0`, `u` slots.
+#'
+#' @param object (`DataDA`)\cr object you want to update.
+#' @param u (`numeric`)\cr the new DLT free survival times for all patients,
+#'   i.e. for existing patients in the `object` as well as for new patients.
+#' @param t0 (`numeric`)\cr the time that each patient starts DLT observation window.
+#'   This parameter covers all patients, i.e. existing patients in the `object`
+#'   as well as for new patients.
+#' @param trialtime (`numeric`)\cr current time in the trial, i.e. a followup time.
```

Comment:
```suggestion
#' @param trialtime (`number`)\cr current time in the trial, i.e. a followup time.
```
as this is only scalar right?